sdcm/cluster_k8s/__init__.py

```diff
@@ -110,6 +110,11 @@
 SCYLLA_CONFIG_NAME = "scylla-config"
 SCYLLA_AGENT_CONFIG_NAME = "scylla-agent-config"
 
+# NOTE: these values are taken from the default values of the "scylla-manager" helm chart.
+#       Needs to be defined separately to be able to reuse for image caching running on local K8S
+SCYLLA_VERSION_IN_SCYLLA_MANAGER = "4.3.0"
```

Comment:
why not read them directly from the helm chart we are gonna use ?

Reply:
It is pain.

Reply:
so it seems. I thought it's a plain public URL. I could only hope...